app/reducers/analytics.jsx

```diff
@@ -17,6 +19,10 @@ const reducer = (state = initialState, action) => {
   case GET_WEEK_DREAMS:
     newState.week = action.dreams
     return newState
+
+  case SET_USER:
+    newState.user = action.user;
+    return newState
```

Comment:
@chloehwang why is set user being repeated here? isnt there a  similar action already on auth?